symengine/visitor.h

```diff
@@ -209,8 +209,8 @@ class CoeffVisitor : public StopVisitor<CoeffVisitor> {
     }
 };
 
-RCP<const Basic> coeff(const Basic &b, const RCP<const Symbol> &x,
-        const RCP<const Integer> &n);
+RCP<const Basic> coef(const Basic &b, const RCP<const Symbol> &x,
+                      const RCP<const Integer> &n);
```

Comment:
It should be called `coeff`, but there is a macro in one of the headers in sage that interferes with this. I only added 27a0db6 as a temporary fix. I'll try to find a way to avoid this.